kitchensink/index.html

```diff
@@ -1144,6 +1143,149 @@ <h2 class="mdl-card__title-text"><i class="fa fa-newspaper-o fa-fw " aria-hidden
+                  </div>
+                </div>
+  </div>
+ 
+  <!-- Controls -->
+  <a class="left carousel-control" href="#bs-carousel" role="button" data-slide="prev">
+    <span class="fa fa-chevron-left"></span>
+  </a>
+  <a class="right carousel-control" href="#bs-carousel" role="button" data-slide="next">
+    <span class="fa fa-chevron-right"></span>
+  </a>
+</div> 
+
+<h3>Using Full Width</h3>
+<div id="bs-carousel-fw" class="carousel slide" data-ride="carousel" data-interval="5000" data-pause="hover" data-keyboard="true">
+    <!-- Indicators -->
+    <ol class="carousel-indicators">
+      <li data-target="#bs-carousel-fw" data-slide-to="0" class="active"></li>
+      <li data-target="#bs-carousel-fw" data-slide-to="1"></li>
+      <li data-target="#bs-carousel-fw" data-slide-to="2"></li>
+      <li data-target="#bs-carousel-fw" data-slide-to="3"></li>
+      <li data-target="#bs-carousel-fw" data-slide-to="4"></li>
+      <li data-target="#bs-carousel-fw" data-slide-to="5"></li>
+      <li data-target="#bs-carousel-fw" data-slide-to="6"></li>   
+    </ol>
+   
+    <!-- Wrapper for slides -->
+    <div class="carousel-inner" role="listbox">
+      <div class="item active">
+        <img class="carousel-full-size" src="http://via.placeholder.com/640x360" alt="image description goes here">
+        <div class="carousel-caption">
+          <h3>Image #1</h3>
+        </div>
+      </div>
+      <div class="item">
+          <img class="carousel-full-size" src="http://via.placeholder.com/640x360" alt="description goes here">
+          <div class="carousel-caption">
+            <h3>Image #2</h3>
+          </div>
+        </div>
+        <div class="item">
+            <img class="carousel-full-size" src="http://via.placeholder.com/640x360" alt="images description goes here">
+            <div class="carousel-caption">
+              <h3>Image #3</h3>
+            </div>
+          </div>
+          <div class="item">
+              <img class="carousel-full-size" src="http://via.placeholder.com/640x360" alt="image description goes here">
+              <div class="carousel-caption">
+                <h3>Image #4</h3>
+              </div>
+            </div>
+            <div class="item">
+                <img class="carousel-full-size" src="http://via.placeholder.com/640x360" alt="image description goes here">
+                <div class="carousel-caption">
+                  <h3>Image #5</h3>
+                </div>
+              </div>
+              <div class="item">
+                  <img class="carousel-full-size" src="http://via.placeholder.com/640x360" alt="image description goes here">
+                  <div class="carousel-caption">
+                    <h3>Image #6</h3>
+                  </div>
+                </div>
+                <div class="item">
+                    <img class="carousel-full-size" src="http://via.placeholder.com/640x360" alt="image description goes here">
```

Comment:
Can we download some placeholder images instead?

Reply:
Yes, good idea, will send this updated commit soon.

Reply:
> Can we download some placeholder images instead?

Ok, placeholder images have been brought in locally to our images folder and added to this PR.